impl.js

```diff
@@ -241,7 +241,7 @@ function generateRequestId() {
   return crypto.randomBytes(7).toString('base64');
 }
 
-function applyContextDefaults(data) {
+function applyContextDefaults(data, contextType) {
   // Add in data to the context
   var context = Object.assign({}, data.context || {});
```

Comment:
Better handle the case where there isn't a context type (i.e. do nothing)

Reply:
What's the best solution here?
My initial thought was to add an else to the if/else if logic inside, but I don't know what it should be.
`context.ortoo.applicationId = 'governorhub'` or something else entirely?